api/src/models/api/stardust/INftDetailsResponse.ts

```diff
@@ -0,0 +1,78 @@
+import { IResponse } from "../IResponse";
+
+export interface INftActivityHistory {
+    /**
+     * transaction Id
+     */
+    transactionId: string;
+    /**
+     * date of transaction
+     */
+    date: string;
+    /**
+     * action
```

Comment:
Can we write some more descriptive JSDocs ? Like this they don't really add any value, just clutter. You can checkout other file for convention.

Reply:
Done